vufs/src/main/java/com/stnetix/ariaddna/vufs/DTO/ChunkFile.java

```diff
@@ -0,0 +1,29 @@
+package com.stnetix.ariaddna.vufs.DTO;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * Describe the chunk of file. Field url contain the place where this chunk stored.
+ * Used in class {@link VirtualFile} as metadata of part of file.
+ */
+public class ChunkFile implements Serializable {
```

Comment:
For history I write message here:

- every chunk should be contain field with hash or CRC code for detect errors in raw data
- for every chunk we can make calculate range bytes from source file
- all field contain ID should be type UUID